net-dns/dnsdist/metadata.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>bgo@9dt.de</email>
+		<name>Vladimir Datsevich</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<longdescription lang="en">
+	</longdescription>
+	<use>
+		<flag name="clang">Use Clang compiler instead of GCC</flag>
+		<flag name="dnscrypt">Enable DNSCRypt support</flag>
+		<flag name="readline">Enable support for libreadline, a GNU line-editing library that almost everyone wants</flag>
+		<flag name="libedit">Use the libedit library (replacement for readline)</flag>
+		<flag name="sodium">Use dev-libs/libsodium for cryptography</flag>
+		<flag name="protobuf">Enable support for dev-libs/protobuf</flag>
```

Comment:
Yes, that is better. However, I would actually wonder if you really ought to name it protobuf when it's about remote logging.